src/citrine/informatics/predictors.py

```diff
@@ -226,8 +228,63 @@ def __str__(self):
         return '<GraphPredictor {!r}>'.format(self.name)
 
 
-class ExpressionPredictor(Serializable['ExpressionPredictor'], Predictor):
-    """[ALPHA] A predictor interface that allows calculator expressions.
+class DeprecatedExpressionPredictor(Serializable['DeprecatedExpressionPredictor'], Predictor):
```

Comment:
Can we add a deprecated annotation here (we use https://pypi.org/project/Deprecated/ )

Reply:
Not to an entire class. (I tried.)

Reply:
We could deprecate the init, but the deprecation message isn't helpful. I figured if a user decided to use `DeprecatedExpressionPredictor` the deprecation was implied.

Reply:
We could rename the class and have a construction function that's deprecated 🤷

Reply:
I just tried using a function but that causes issues with our serialization logic.

Reply:
When I had to do this for `InorganicDescriptor`, I raised a warning: https://github.com/CitrineInformatics/citrine-python/blob/master/src/citrine/informatics/descriptors.py#L116

Reply:
I remembered that it only worked for one of {functions, classes}, but I thought it was classes.  Sorry.  Explicitly raising the warning like in the example @bfolie referenced should do it.